templates/base.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>{% block title %}Taxi Service{% endblock %}</title>
+    {% load static %}
+    <link rel="stylesheet" href="{% static 'css/styles.css' %}">
+</head>
+<body>
```

Comment:
The `<body>` tag should include a block for `sidebar` as specified in the task description. Consider adding `{% block sidebar %}{% endblock %}` to allow child templates to override this block if needed.